testing/fzf/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Kevin Daudt <ops@ikke.info>
+# Maintainer: Kevin Daudt <ops@ikke.info>
+pkgname=fzf
+_pkgver="0.17.0-2"
+pkgver=${_pkgver/-/.}
+pkgrel=0
+pkgdesc="A command-line fuzzy finder"
+url="https://github.com/junegunn/fzf"
+arch="all"
+license="MIT"
+depends="go"
```

Comment:
is `go` a buildtime or runtime dependency?

Reply:
You are right, go is only a makedepends.